android/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 5.5.1
+version: 5.5.2
```

Comment:
5.5.2 was released [in June already](https://github.com/tidev/ti.map/releases/tag/v5.5.2-android). Something is broken here. I will bump it to 5.5.3, but it seems like the last release was not properly tagged

Reply:
ok ok. It was distributed with the SDK so that was fine at least. I'll add a PR to the SDK to include those modules

Reply:
Already done :)